pkg/network/http/batch_manager.go

```diff
@@ -84,7 +84,11 @@ func (m *batchManager) GetTransactionsFrom(notification httpNotification) ([]htt
 	state.idx = int(notification.batch_idx) + 1
 	state.pos = 0
 
-	return batch.Transactions()[offset:], nil
+	txns := make([]httpTX, len(batch.Transactions()[offset:]))
+	for idx, tx := range batch.Transactions()[offset:] {
+		txns[idx] = &tx
+	}
+	return txns, nil
```

Comment:
This change seems like it may have been accidentally included? (Since it's modifying ebpf code)